UnityProject/Assets/Scripts/Objects/Research/Artifacts/Effects/AreaEffects/StunAreaEffect.cs

```diff
@@ -0,0 +1,19 @@
+using HealthV2;
+using UnityEngine;
+
+namespace Systems.Research
+{
+	[CreateAssetMenu(fileName = "StunAreaEffect", menuName = "ScriptableObjects/Systems/Artifacts/StunAreaEffect")]
+	public class StunAreaEffect : AreaEffectBase
+	{
+		public int StunDuration = 3;
+		public bool DropItems = false;
+		public bool ArmourBlockable = false;
+		public bool StopMovement = true;
+
+		public override void OnEffect(PlayerScript player, BodyPart part = null)
+		{
+			player.GetComponent<RegisterPlayer>().ServerStun(StunDuration, DropItems, ArmourBlockable, StopMovement);
```

Comment:
```suggestion
			player.RegisterPlayer.ServerStun(StunDuration, DropItems, ArmourBlockable, StopMovement);
```
Should be a thing

Reply:
Unfortunately is not

Reply:
`player.registerTile` is indeed what you're after:
![intellisense](https://i.imgur.com/5JGEGHE.png)

```suggestion
			player.registerTile.ServerStun(StunDuration, DropItems, ArmourBlockable, StopMovement);
```

Reply:
god dammit I saw that and thought it was wrong, was looking for a registerPlayer field

Reply:
Yeah, terrible names 😅